client/signup/steps/woocommerce-install/transfer/index.tsx

```diff
@@ -52,13 +52,13 @@ export default function Transfer( props: WooCommerceInstallProps ): ReactElement
 				<>
 					{ isAtomic && (
 						<InstallPlugins
-							onFailure={ () => handleTransferFailure( 'install ' ) }
+							onFailure={ ( type ) => handleTransferFailure( type ) }
```

Comment:
@Rolilink I looked into moving these to a separate file, but we're also handling `hasFailed` state in `handleTransferFailure`. That would still need to be passed down somehow. Let me know if I missed something or if you have other thoughts.